redun/executors/gcp_utils.py

```diff
@@ -27,28 +27,29 @@ def get_gcp_client(
 ) -> Union[batch_v1.BatchServiceClient, batch_v1.BatchServiceAsyncClient]:
     return batch_v1.BatchServiceClient() if sync else batch_v1.BatchServiceAsyncClient()
 
+def gb_to_mib(gb):
+    # Convert GiB to MiB.
+    return int(gb * 1024)
 
 def batch_submit(
     client: Union[batch_v1.BatchServiceClient, batch_v1.BatchServiceAsyncClient],
     job_name: str,
     project: str,
     region: str,
-    mount_buckets: List[str],
-    gcs_scratch_prefix: str,
```

Comment:
huh why are they removing the `gcs_scratch_prefix`

Reply:
It was actually not explicitly used by the function